app/services/services.rb

```diff
@@ -17,13 +17,23 @@ def self.service_name_options
   end
 
   def self.service_cis_options
-    load_values(:cis).map do |cis|
-      { id: cis[:id], label: "#{cis[:name]} - #{cis[:address]}" }
+    unless Rails.env.test? 
```

Comment:
Trailing whitespace detected.<br>Do not use `unless` with `else`. Rewrite these with the positive case first.